source/_integrations/octoprint.markdown

```diff
@@ -77,15 +77,7 @@ Supported sensors:
 
 ## Camera
 
-If the OctoPrint host is equipped with a web camera it is possible to add this as well.
-
-```yaml
-camera:
-  - platform: mjpeg
-    name: OctoPrint
-    still_image_url: http://YOUR_OCTOPRINT_HOST_IP/webcam/?action=snapshot
-    mjpeg_url: http://YOUR_OCTOPRINT_HOST_IP/webcam/?action=stream
-```
+If the OctoPrint host is equipped with a web camera it is possible to add this as well using the [`MJPEG IP Camera`](/integrations/mjpeg) integration. Use `http://YOUR_OCTOPRINT_HOST_IP/webcam/?action=stream` for the MJPEG URL and `http://YOUR_OCTOPRINT_HOST_IP/webcam/?action=snapshot` as the still image URL.
```

Comment:
Is it possible (or desirable) to add a link for adding the integration directly? 🤔 Similar to the blue "ADD INTEGRATION" button 👇 but for the MJPEG IP Camera integration.

<img width="735" alt="image" src="https://user-images.githubusercontent.com/1417619/161395435-98fba8a0-a627-4d67-b726-fc1fdc05144e.png">

Reply:
Reverse engineering the include gave me this:

```
{% my config_flow_start badge domain=page.ha_domain %}
```

So I guess something like

```
{% my config_flow_start badge domain=mjpeg %}
```

would work. Then the question is only if it's the right thing to do 🙂

Reply:
Added it now to test if it works, but I can revert it if it's considered more confusing/harmful than anything else.